Runtime/Triangulator.cs

```diff
@@ -1734,9 +1734,8 @@ private struct RefineMeshStep
             {
                 public readonly T2 Center;
                 public readonly T RadiusSq;
-                private readonly T offset;
```

Comment:
The offset in the `Circle` serves a specific purpose: it improves alignment and enhances vectorization, leading to better overall performance. As a result, this structure for `T = float` has the same size as `float4`, and as `float3` without the offset.

You can find online benchmarks comparing `float3` and `float4`. In tests I've conducted in the past, the performance improvement was indeed measurable.